File: cpp/main/stage1/Stage1.cpp
/**
 * Uncomment the tests, and make them pass
 */

#include "Stage1.hpp"

const Stage1 Stage1::POTATO = Stage1(1);
const Stage1 Stage1::TOMATO = Stage1(2);
const Stage1 Stage1::LETTUCE = Stage1(2);
const Stage1 Stage1::APPLE = Stage1(1);
const Stage1 Stage1::BANANA = Stage1(3);
const Stage1 Stage1::PEER = Stage1(2);

int Stage1::getPrice() const {
  return price;
}

File: typescript/src/stage1/Stage1.ts
export enum Item {
  POTATO = 1,
  TOMATO = 2,
  LETTUCE = 2,
  APPLE = 1,
  BANANA = 3,
  PEER = 2,
}

//todo Uncomment the tests, and make them pass

export class Stage1 {
  getPrice(item: Item) {
    return item;
  }
}
